ground/src/main/java/com/google/android/ground/model/geometry/MultiPolygon.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.geometry
+
+import com.google.common.collect.ImmutableList
+
+/** A collection of [Polygon]s.*/
+data class MultiPolygon(val polygons: List<Polygon>) : Geometry {
+    // TODO: Enforce definitional constraints on initialization: polygons may not overlap and can only touch at single points.
+
+    /** Returns the first coordinate of the first polygon in this multipolygon. */
+    override val coordinate: Coordinate = polygons.first().coordinate
```

Comment:
What is this used for? Is it needed?

Reply:
Yes, we'll need coordinates implementations for our Geometry types since our Firestore conversion routines and other parts of the codebase rely on them. This is modeled directly after JTS. e.g. see `GeometryConverter.kt`

Reply:
What would `coordinate` `MultiPolygon` and `Polygon` be used for? If we don't need it we should leave it out. Afaik the conversion routines work differently for different geometry types (e.g., for `Point` we just look for a single set of coordinates, for `Polygon` we look at the outer shell and inner shells, etc.)